README.md

```diff
@@ -1,5 +1,21 @@
 # v2x-ros-conversion
 
+The V2X-ROS-Conversion package contains two ROS nodes origionally developed and packaged inside the CARMA Platform communication stack: Message node and J2735 Convertor Node. These two packages were ported to a seperate repository to allow for other projects to utilize these ROS nodes.
```

Comment:
Let's update this README to mention only the functionality in the repository, we can remove the reference to CARMA Platform.

We should also add a note about the ros2 version being used here.